docs/whats-new.rst

```diff
@@ -27,6 +27,12 @@ v2021.xx (unreleased)
 
 .. _whats-new.2021.xx:
 
+
+Breaking changes
```

Comment:
Wonder if this should be a breaking change or bugfix. Really just semantics, though.